contracts/cw20-base/helpers.ts

```diff
@@ -20,24 +20,29 @@ interface Options {
   readonly httpUrl: string
   readonly networkId: string
   readonly feeToken: string
-  readonly gasPrice: number
+  readonly gasPrice: GasPrice
```

Comment:
This changed. Is this using 0.22 still? or 0.23?

Reply:
This is for v0.23.0